src/main/java/nextstep/ladder/domain/Ladder.java

```diff
@@ -15,6 +14,13 @@ public Ladder(Height height, int countOfPerson) {
                 .collect(Collectors.toList());
     }
 
+    public int climb(int position) {
+        for (Line line : lines) {
+            position += line.move(position).getValue();
+        }
+        return position;
+    }
```

Comment:
```suggestion
    public Direction climb(int position) {
        Direction direction = Direction.of(position);
        for (Line line : lines) {
            direction = direction.plus(line.move(position));
        }
        return direction;
    }
```

많은 부분 수정해야 할지도 모르지만, 저는 이렇게 도전해볼것 같네요 🤔

Reply:
Direction enum은 -1(왼쪽), 0(이동 X), 1(오른쪽) 방향을 의미하는 값을 가지고 있습니다.
climb 메소드 에서 반환하는 값은 각 사용자의 첫번째 위치 값을 받아 사다리 결과의 위치 값을 반환 하는값으로 의도하였습니다!

반환값은 그대로 유지하되 getValue()로 값을 꺼내지 않고 객체한테 물어보는 식으로 코드를 구현했는데 해당 방법은 어떤지요..? 이 방법도 별로면 리뷰어님이 알려주신대로 해보겠습니다!! 😥

※ 자세한 내용은 commit 내용 확인 부탁 드립니다..!

```java
    public int climb(int position) {
        for (Line line : lines) {
            position = line.move(position).move(position);
        }
        return position;
    }
```